content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -2,77 +2,77 @@
 Assign leads based on scoring
 =============================
 
-With *Leads Scoring* you can automatically rank your leads based on
-selected criterias.
+With *Lead Scoring* you can automatically rank your leads based on selected criteria.
 
-For example you could score customers from your country higher or the
-ones that visited specific pages on your website.
+For example, you could score customers higher if coming from the same country as your
+company over a customer from a different country.
 
 Configuration
 =============
 
-To use scoring, install the free module *Lead Scoring* under your
-*Apps* page (only available in Odoo Enterprise).
+To use scoring, install the free module *Lead Scoring* under your *Apps* page (only available in
+Odoo Enterprise).
 
-.. image:: media/lead_scoring01.png
+.. image:: lead_scoring/lead-scoring-module.png
    :align: center
+   :alt: Lead Scoring module installation
 
 Create scoring rules
 ====================
 
-You now have a new tab in your *CRM* app called *Leads Management*
-where you can manage your scoring rules.
+You now have a new tab in your *CRM* app called *Leads* where you can manage your scoring rules.
```

Comment:
For menu paths, let's stay consistent and use `menuselection` (see content guidelines, the part about the structure)
```suggestion
To manage your scoring rules, go to :menuselection:`CRM --> Leads --> Scoring Rules`.
```